pypesto/optimize/optimizer.py

```diff
@@ -558,3 +563,50 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, options: Dict = None):
```

Comment:
```suggestion
    def __init__(self, par_sigma0: float = 0.25, options: Dict = None):
```

Reply:
I implemented it, although it seems to do nothing new